quartz/src/main/java/org/quartz/core/QuartzSchedulerMBeanImpl.java

```diff
@@ -938,9 +938,9 @@ public MBeanNotificationInfo[] getNotificationInfo() {
      * @see javax.management.NotificationBroadcaster#addNotificationListener(javax.management.NotificationListener,
      *      javax.management.NotificationFilter, java.lang.Object)
      */
-    public void addNotificationListener(NotificationListener notif,
+    public void addNotificationListener(NotificationListener listener,
```

Comment:
Sometimes `notif` was used for `notification` and sometimes for `listener`. By not abbreviating, things are clearer...